java/src/Ice/src/main/java/com/zeroc/Ice/LoggerI.java

```diff
@@ -4,6 +4,9 @@
 
 package com.zeroc.Ice;
 
+/**
+ * @hidden Kept public because it's used by IceGridGUI.
+ */
```

Comment:
No clear why IceGridGUI reuses this logger. It would be better that it implements its own and kept this private.